python/tank/util/pyside6_patcher.py

```diff
@@ -253,6 +253,17 @@ def versionFunctions(self, version_profile=None):
 
         QtGui.QOpenGLContext.versionFunctions = versionFunctions
 
+    @classmethod
+    def _patch_QWheelEvent(cls, QtGui):
+        """Patch QWheelEvent."""
+
+        def delta(self):
+            """Patch the delta method."""
+
+            return self.angleDelta().y()
```

Comment:
I would just add a comment here to explain why we patch the `delta` with `QWheelEvent::angleDelta().y()`
```suggestion
            # Use the more common mouse vertical scroll as the delta.
            # Horizontal scroll is ignored, use angleDelta().x() if the horizontal scroll is needed.
            return self.angleDelta().y()
```